.github/workflows/pull-request.yml

```
@@ -21,9 +24,37 @@ jobs:
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install Dependencies
+        run: npm ci
+
+      - name: Run Tests
+        run: npm test
+        working-directory: ${{ github.workspace }}
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)